_components/form-blocks.md

```diff
@@ -107,7 +107,6 @@ title: Forms Blocks
     <p>As you customize this form, ensure it continues to:</p>
     <ul>
       <li>Label the optional ones. Users can infer that all the others are required.</li>
-      <li>Let users type their state's abbreviation when they reach the "state" drop down menu.</li>
       <li>Support both 5 and 9 digit zips. Some addresses require a nine digit ZIP code. The input mask should be "#####-####"" so that the text is properly formatted, regardless of whether a user enters a 5 or 9 digit zip code.</li>
```

Comment:
Should we refer to the note in accessibility about Politespace? Or is that junking up the content too much?